input/pagecontent/reference.md

```diff
@@ -1541,6 +1541,63 @@ If `Usage` is unspecified, the default is `#example`.
   * identifier[NPI].value = "8274017284"
   ```
 
+  This instance would be incorporated into a Composition with a statement such as:
+
+  ```
+  *  section[0].entry[1] = Reference(DrDavidAnydoc)
```

Comment:
`Composition.section.entry` is a _reference_ (as seen in this line).  So the real inlining needs to be in the `contained` element.  In fact, this example is incomplete as it is because you must also have:
```
* contained[0] = DrDavidAnydoc
```
otherwise the `Reference(DrDavidAnydoc)` will be exported as `Practitioner/DrDavidAnydoc` rather than the contained reference: `#DrDavidAnydoc`.

Reply:
If you really want an example that does not use `contained` then you should use `Bundle.entry.resource`.